edgelet/iotedge/src/check/checks/container_resolve_parent_hostname.rs

```diff
@@ -49,27 +55,33 @@ impl ContainerResolveParentHostname {
             return Ok(CheckResult::Skipped);
         };
 
-        let parent_hostname = if let Some(hub_hostname) = settings.parent_hostname() {
-            hub_hostname.to_string()
-        } else {
-            return Ok(CheckResult::Skipped);
-        };
+        let mut args = vec!["run".to_owned(), "--rm".to_owned()];
+
+        settings
+            .agent()
+            .config()
+            .create_options()
+            .host_config()
+            .and_then(docker::models::HostConfig::extra_hosts)
+            .iter()
+            .for_each(|extra_hosts| {
+                extra_hosts
+                    .iter()
+                    .for_each(|host| args.push(format!("--add-host={}", host)))
```

Comment:
I suppose this is why you need to uncomment extraHosts above?

Reply:
Correct :)